controllers/nugus_controller/nugus_controller_test.cpp

```diff
@@ -169,11 +173,13 @@ int main(int argc, char** argv) {
             current_num += 1;
             msg.set_num(current_num);
 
-            N = msg.ByteSizeLong();
-            data.resize(N);
-            msg.SerializeToArray(data.data(), N);
+            Nh = msg.ByteSizeLong();
+            data.resize(Nh);
+            msg.SerializeToArray(data.data(), Nh);
+
+			Nn = htonl(Nh);
```

Comment:
```suggestion
            // Covert to host endianness, which might be different to network endianness
            Nn = htonl(Nh);
```

Reply:
This is plain wrong.
```suggestion
            // Covert to network endianness, which might be different to network endianness
            Nn = htonl(Nh);
```

Reply:
Please elaborate

Reply:
Not that it makes any difference :P `htonl` and `ntohl` (host to network long, and network to host long) are exactly the same function :P